include/solvers/multiphysics_interface.h

```diff
@@ -65,6 +65,34 @@ class MultiphysicsInterface
     return active_physics;
   }
 
+  /**
+   * @brief Write physic solved in the terminal
+   *
+   * @param physics_id number associated with auxiliary physics in multiphysics.h
+   */
+  void
+  annonce_physics(const PhysicsID physics_id)
+  {
+    if (physics_id == 1)
```

Comment:
Instead of using the raw numbers for the physics id, it's better to use the enum from the multiphysics namespace. This way things are coherent everywhere. 
E.g.  https://github.com/lethe-cfd/lethe/blob/master/include/core/multiphysics.h
PhysicsID::fluid_dynamics for example.

Reply:
Oh yes indeed! Thank you :)